ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -7181,6 +7181,20 @@ module SDN_controller = struct
     let uuid = List.assoc "uuid" params in
     let ref = Client.SDN_controller.get_by_uuid rpc session_id uuid in
     Client.SDN_controller.forget rpc session_id ref
+
+  let open_port printer rpc session_id params =
+    let uuid = List.assoc "uuid" params in
+    let ref = Client.SDN_controller.get_by_uuid rpc session_id uuid in
+    let protocol = Record_util.sdn_port_protocol_of_string (List.assoc "protocol" params) in
+    let port = Int64.of_string (List.assoc "port" params) in
+    Client.SDN_controller.open_port rpc session_id ref protocol port
+
+  let close_port printer rpc session_id params =
+    let uuid = List.assoc "uuid" params in
+    let ref = Client.SDN_controller.get_by_uuid rpc session_id uuid in
+    let protocol = Record_util.sdn_port_protocol_of_string (List.assoc "protocol" params) in
+    let port = Int64.of_string (List.assoc "port" params) in
+    Client.SDN_controller.close_port rpc session_id ref protocol port
```

Comment:
I suspect you need to run `make format` for this section

Reply:
Probably, I always run it after I'm done with my dev. :)

Reply:
I'm wondering, are these modifications necessary?
I'm not sure I understand the uses of this file since the method is not available in `xe` cli.

and if necessary, should I iter through all the pools host like I do in `message_forwarding.ml`?